src/_gettsim/wohngeld/miete.py

```diff
@@ -230,11 +213,7 @@ def miete_m_hh_mit_baujahr(
         wohnen__baujahr_immobilie_hh,
         side="left",
     )
-    max_miete_m = max_miete_m_lookup.lookup_table[
-        selected_bin_index,
-        anzahl_personen_hh - max_miete_m_lookup.lookup_base_to_subtract_rows[selected_bin_index],
-        mietstufe - max_miete_m_lookup.lookup_base_to_subtract_cols[selected_bin_index],
-    ]  # fmt: skip
+    max_miete_m = max_miete_m_lookup.lookup_table.lookup(selected_bin_index,anzahl_personen_hh,mietstufe)  # fmt: skip
```

Comment:
```suggestion
    max_miete_m = max_miete_m_lookup.lookup_table.lookup(baujahr_index,anzahl_personen_hh,mietstufe)  
```
- Rename variable above, too
- We just skipped that because the old line became very ugly otherwise, not an issue anymore!